app/assets/stylesheets/layout_components/forms.scss

```diff
@@ -412,6 +412,35 @@ input[type="checkbox"] {
     margin-bottom: $base-margin;
   }
 }
+/* -------------- File Uploader ---------- */
+
+.uploader {
+  &__file {
+    border-bottom: $base-border;
+    padding: $base-padding/2 0;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__icon {
+    font-size: $base-font-size*1.2;
+    color: $charcoal;
+
+    &-clear {
+      @extend .uploader__icon;
+      cursor: pointer;
+
+      &:hover {
+        color: $brand-red;
+      }
+    }
+
+    &-doc {
```

Comment:
same thing here, also I don't like naming a modifier after the type of icon that we're using (what if we decide to have different icons for pdf, png, etc. in the future?). something like --spacing might be better?